msg/ekf2_innovations.msg

```diff
@@ -1,19 +1,34 @@
 uint64 timestamp		# time since system start (microseconds)
-float32[6] vel_pos_innov	# velocity and position innovations
-float32[3] mag_innov 		# earth magnetic field innovations
-float32 heading_innov 		# heading innovation
-float32 airspeed_innov		# airspeed innovation
-float32 beta_innov		# synthetic sideslip innovation
-float32[2] flow_innov   	# flow innovation
-float32 hagl_innov      	# innovation from the terrain estimator for the height above ground level measurement  (m)
-float32[6] vel_pos_innov_var 	# velocity and position innovation variances
-float32[3] mag_innov_var	# earth magnetic field innovation variance
-float32 heading_innov_var 	# heading innovation variance
-float32 airspeed_innov_var	# Airspeed innovation variance
-float32 beta_innov_var		# synthetic sideslip innovation variance
-float32[2] flow_innov_var	# flow innovation variance
-float32 hagl_innov_var          # innovation variance from the terrain estimator for the height above ground level measurement (m^2)
-float32[3] output_tracking_error # return a vector containing the output predictor angular, velocity and position tracking error magnitudes (rad), (m/s), (m)
-float32[2] drag_innov		# drag specific force innovation (m/s/s)
-float32[2] drag_innov_var	# drag specific force innovation variance (m/s/s)**2
-float32[2] aux_vel_innov	# auxiliary NE velocity innovations from landing target measurement (m/s)
+
+# GPS
+float32[2] gps_hvel	# horizontal GPS velocity innovation
+float32    gps_vvel	# vertical GPS velocity innovation
+float32[2] gps_hpos	# horizontal GPS position innovation
+float32    gps_vpos	# vertical GPS position innovation
+
+# External Vision
+float32[2] ev_hvel	# horizontal external vision velocity innovation
+float32    ev_vvel	# vertical external vision velocity innovation
+float32[2] ev_hpos	# horizontal external vision position innovation
+float32    ev_vpos	# vertical external vision position innovation
+
+# Height sensors
+float32 rng_vpos	# range sensor height innovation
+float32 baro_vpos	# barometer height innovation
+
+# Auxiliary (horizontal) velocity
+float32[2] aux_hvel	# horizontal auxiliar velocity innovation from landing target measurement
+float32[2] flow		# flow innvoation
+
+# Various
+float32 heading		# heading innovation
+float32[3] mag		# earth magnetic field innovation
+float32[2] drag		# drag specific force innovation
+float32 airspeed	# airspeed innovation
+float32 beta		# synthetic sideslip innovation
+float32 hagl		# height of ground innovation
+
+# TOPICS ekf2_innovations ekf2_innovation_variances ekf2_innovation_test_ratios
+
+# For the innovation test ratios are scalar values. In case the field is a vector,
```

Comment:
Please bring this above the `#TOPICS` line (let the `#TOPICS` line be the last one.